api/shared/nodenetworkconfigurationenactment_types.go

```diff
@@ -13,10 +14,34 @@ type NodeNetworkConfigurationEnactmentStatus struct {
 	// the policy desiredState as template
 	DesiredState State `json:"desiredState,omitempty"`
 
+	// CaptureStatus The capture status after processing the capture at NNCP
+	CaptureStatus NodeNetworkConfigurationEnactmentCaptureStatus `json:"captureStatus,omitempty"`
+
 	// The generation from policy needed to check if an enactment
 	// condition status belongs to the same policy version
-	PolicyGeneration int64         `json:"policyGeneration,omitempty"`
-	Conditions       ConditionList `json:"conditions,omitempty"`
+	PolicyGeneration int64 `json:"policyGeneration,omitempty"`
+
+	Conditions ConditionList `json:"conditions,omitempty"`
+}
+
+type NodeNetworkConfigurationEnactmentCaptureStatus struct {
```

Comment:
The way it is structured now, it looks like the name should specify the "cache" and not the capture specifically.

In `nmpolicy`, "cache" was a generalization of the input, which consists only of captures.
So we can either keep the generalization here as well, and call this CachedState, and in it specify the captures, or just do not generalize it and leave only one "level".

BTW, we are talking about "state" and not "status", right?

Reply:
It is not just the cache, The field contain both the capture cache and the metainfo of the last call to the `nmpolicy.GenerateState`.
I didn't want to use the term state since we use it in the enactment to represent yamls (desiredState, capture.state).